dev/devicelab/bin/tasks/flavors_test.dart

```diff
@@ -13,8 +13,9 @@ Future<void> main() async {
   await task(() async {
     await createFlavorsTest().call();
     await createIntegrationTestFlavorsTest().call();
-    // test install and uninstall of flavors app
-    await inDirectory('${flutterDirectory.path}/dev/integration_tests/flavors', () async {
+
+    final TaskResult installTestsResult = await inDirectory(
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#if-you-have-a-newline-after-some-opening-punctuation-match-it-on-the-closing-punctuation

On the second line (18) here, the indent should just be 2 more than line 17, and then the start of the callback should be on a new line (19) indented the same as line 18, and then the rest of the callback body indented +2.

Reply:
Agreed. It should be fixed now. Fun fact: I got the original formatting from `dart format`